Develop

victor-buldenko opened the pull request:
[DEMO LINK](https://victor-buldenko.github.io/Museum_2/).

AlisaVlasova reviewed (changes requested):
хедер трошки перекриває адресу
<img width="287" alt="image" src="https://user-images.githubusercontent.com/66749919/205436906-e751258c-f78b-4970-bc63-90a7f03b41e6.png">

polosanya reviewed (changes requested):
Looks really great! 🔥 
Just a small comment:
1. Add a cursor pointer to the button so the user can understand that it's interactive. 
Also, it's a bit confusing that it's grey until email matches the right format, but fixing it is up to you.
![image](https://user-images.githubusercontent.com/96409563/209195374-92678ddb-e789-4493-b0c5-fbafb754d8b0.png)

DarkMistyRoom reviewed (changes requested):
- 
![image](https://user-images.githubusercontent.com/98764641/209320824-225cc43f-ec94-42cc-8df4-12ba9fcf8c44.png)

- make the cursor a pointer over this select
![image](https://user-images.githubusercontent.com/98764641/209320950-ea00d353-8f10-4d71-b332-f25f9ae85e83.png)

- what is it? I can't scroll
![image](https://user-images.githubusercontent.com/98764641/209321147-3050b21e-2a15-437a-9a35-2fc3ec4b6568.png)

- check your breakpoints
![image](https://user-images.githubusercontent.com/98764641/209321783-3514ee53-507a-43b5-b534-8715e268e9bc.png)

- ooops)
![image](https://user-images.githubusercontent.com/98764641/209321836-69b1f52c-1083-42a3-849f-a9af0e19fc19.png)

- fix the cursor
https://www.loom.com/share/c450ad0453f24ef68554a74f80a9067a

DarkMistyRoom reviewed (changes requested):
- It still doesn't look properly on the iPad Air
![image](https://user-images.githubusercontent.com/98764641/210217632-7ee3c716-ebd1-4b49-89ed-ca5b2b0af552.png)

- The header should be fit the window height on the any screen
![image](https://user-images.githubusercontent.com/98764641/210217697-4b97e186-9c00-4744-b11a-9fa6e8a4d86c.png)

- this section looks different on the mockup for the tab
![image](https://user-images.githubusercontent.com/98764641/210217843-e0f791ac-8e99-4960-8f6f-521a42bd7a41.png)

- not fixed
![image](https://user-images.githubusercontent.com/98764641/210217891-9b5c78fb-0184-4948-a072-bddb367ee185.png)

- check indents in the footer on the mobile

victor-buldenko commented:
about iphone SE: display height less ~100px of figma template, so start page more than 1viewport. Scroll on.

DarkMistyRoom reviewed (changes requested):
- the page shouldn't have 2 scroll bars
![image](https://user-images.githubusercontent.com/98764641/210235545-937b3989-ad10-4fee-8cec-579a936a54d2.png)

- it still looks different on the tab. be careful
![image](https://user-images.githubusercontent.com/98764641/210235700-7b4bde97-86e6-4278-a5b5-29026f2a7880.png)
![image](https://user-images.githubusercontent.com/98764641/210235756-b0d8c2e0-5471-41f9-99b6-b7b06165b44f.png)

- remember about gaps and use one-direction margins
![image](https://user-images.githubusercontent.com/98764641/210236036-ad15c652-54c4-4d7e-9363-a9bc8c041171.png)

DarkMistyRoom reviewed (changes requested):
Much better! few comments:
- here should be equal indents

![image](https://user-images.githubusercontent.com/98764641/214833641-d4cd6a39-0672-424b-b681-819dfe85628e.png)

- check these indents
![image](https://user-images.githubusercontent.com/98764641/214833950-98a46242-bc83-44ff-bca6-39d4ff7d02ba.png)

loralevitska reviewed (changes requested):
I am impressed with the animation, you did a nice job. It remains to correct a little:
1. Main picture doesn't match with design 
![image](https://user-images.githubusercontent.com/70009205/215256789-a76efbbe-2395-4437-92be-b77044a54c0a.png)
2. Fix all indent with this picture for all modes 
![image](https://user-images.githubusercontent.com/70009205/215256962-aa17657c-d8f0-4dc2-af8a-e33fc202146f.png)
3. This picture is cropped
![image](https://user-images.githubusercontent.com/70009205/215257005-a5fd1ff3-e35f-4ba2-b8e6-1e9a00a08335.png)
4. Button color doesn't match with design 
![image](https://user-images.githubusercontent.com/70009205/215257056-b0a1beeb-3d9f-49ee-8865-1840a3e4c331.png)
5. Add this element
![image](https://user-images.githubusercontent.com/70009205/215257103-8336c53e-eff4-46c0-ab75-0bed05fbe1b1.png)

victor-buldenko commented:
> ~It remains to correct a little:
> 
> *1.  fixed
> *2.  problem was fixed by 1 item from issue list, i guess... correctly i hope )))
> *3. added alter image (<pictures> added)
> *4. Button color doesn't match with design
    ***because button change color when user typed email (screen)
![photo_2023](https://user-images.githubusercontent.com/114029531/215290135-6bd3caa0-c40a-4ce2-8e5f-6f68a1271d8d.jpg)
> *5. Scroll arrow is added

qoa11a reviewed (changes requested):
<img width="1470" alt="image" src="https://user-images.githubusercontent.com/102293862/215317601-e3c3990a-d261-4abb-8722-d63962b76f5c.png">
button color doesn't match the design

victor-buldenko commented:
> <img alt="image" width="1470" src="https://user-images.githubusercontent.com/102293862/215317601-e3c3990a-d261-4abb-8722-d63962b76f5c.png">
> 
> button color doesn't match the design

https://github.com/mate-academy/Museum_2/pull/229#pullrequestreview-1228060858


**You against  each other.
button style is correct, on my & Oleksander Polotnik opinion, because follow button may be interactive option.
Midst 5 reviewers, you first who don`t like this thing
What i have to do about that?**

SanyaBratashchuk reviewed (approved):
LGTM
